### PR TITLE
Update spinner colours

### PR DESCRIPTION
**Pull request by peppy**
Still may require a few more tweaks, but feeling a lot better already.

### CODE REVIEW

File: osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs [hunk 1]
```diff
@@ -44,8 +30,8 @@ private void layout()
                     EdgeEffect = new EdgeEffect
                     {
                         Type = EdgeEffectType.Glow,
-                        Radius = 20,
-                        Colour = glowColour,
+                        Radius = 10,
+                        Colour = Color4.Gray.Opacity(0.2f),
```
Comment (smoogipooo) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: Sure you want to do this instead of using OsuColour?

Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: either way it's just specifying a specific colour. i'm not sure of the difference

Comment (smoogipooo) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: Makes this not skinnable by way of skinning colours.

Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: if it was using osucolour it's just going to be using the gray generation function. which i don't think will ever be skinnable

Comment (smoogipooo) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: Or define a new SpinnerTickGlow colour like there is with SpinnerBase and SpinnerFill with the default being `SpinnerTickGlow = Gray(128)`, or perhaps even referencing one of the existing grays, `OsuColour.Gray8` is close enough imho.

Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: if you insist, but i can guarantee the grays will never be used for skinning

Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: actually i'm not changing it. opacity is also being applied. it's pointles

Comment (smoogipooo) [inline osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs]: I don't see how that changes anything, but whatever... Guess we'll change it when it inevitably comes up.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**